Multisig/Logic/Models/PrivateKey/web3swift/Base58.swift

```diff
@@ -115,7 +115,6 @@ struct Base58 {
 }
 
 
```

Comment:
Does it show two lines here?

Reply:
there are actually two lines. Will remove in the next pr